File: xposed-api-stubs/src/main/java/de/robv/android/xposed/callbacks/XC_LoadPackage.java
package de.robv.android.xposed.callbacks;

import de.robv.android.xposed.XposedBridge;

public abstract class XC_LoadPackage extends XCallback {
    public XC_LoadPackage() {
        super();
    }

    public XC_LoadPackage(int priority) {
        super(priority);
    }

    public abstract void handleLoadPackage(LoadPackageParam lpparam) throws Throwable;

    public static final class LoadPackageParam extends XCallback.Param {
        public String packageName;
        public String processName;
        public ClassLoader classLoader;
        public Object appInfo;
        public boolean isFirstApplication;

        public LoadPackageParam(XposedBridge.CopyOnWriteSortedSet<XC_LoadPackage> callbacks) {
        }
    }
}

File: app/src/main/java/de/robv/android/xposed/callbacks/XC_LoadPackage.kt
package de.robv.android.xposed.callbacks

import android.content.pm.ApplicationInfo

open class XC_LoadPackage {
    class LoadPackageParam {
        var packageName: String? = null
        var processName: String? = null
        var classLoader: ClassLoader? = null
        var appInfo: ApplicationInfo? = null
        var isFirstApplication: Boolean = false
    }
}
